### PR TITLE
[DOCS] Add Beats config example for ingest pipelines

### PR DESCRIPTION
**Pull request by jrodewig**
The Elasticsearch ingest pipeline docs cover pipelines for Fleet and
Elastic Agent. However, the docs don't cover Beats. This adds 'em.

Relates to elastic/beats#28239.

### Preview
https://elasticsearch_78633.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/ingest.html#pipelines-for-beats

### CODE REVIEW

File: docs/reference/ingest.asciidoc [hunk 1]
```diff
@@ -271,6 +271,21 @@ Use the <<index-final-pipeline,`index.final_pipeline`>> index setting to set a
 final pipeline. {es} applies this pipeline after the request or default
 pipeline, even if neither is specified.
 
+[discrete]
+[[pipelines-for-beats]]
+=== Pipelines for {beats}
+
+To add an ingest pipeline to an Elastic Beat, specify the `pipeline`
+parameter under `output.elasticsearch` in `<BEAT_NAME>beat.yml`. For example,
```
Comment (dedemorton) [inline docs/reference/ingest.asciidoc:279]:
```suggestion
parameter under `output.elasticsearch` in `<BEAT_NAME>.yml`. For example,
```
```suggestion
parameter under `output.elasticsearch` in `<BEAT_NAME>beat.yml`. For example,
```

Comment (dedemorton) [reply docs/reference/ingest.asciidoc:279]: Unless you use this somewhere else, I'd just shorten it.

**Review by dedemorton — COMMENTED** [review]
Very minor suggestion; otherwise LGTM. Thanks for doing this!

File: docs/reference/ingest.asciidoc (hunk 1 above)
Comment (jrodewig) [reply docs/reference/ingest.asciidoc]: Great suggestion. Thanks @dedemorton!

**Pull request merged**